crates/client/gateway/src/client/methods.rs

```diff
@@ -190,6 +205,12 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    #[ignore]
```

Comment:
is this still useful?

Reply:
Just serves as a handy stub which we can uncomment to set up fgw tests. I'm in agreement its not very clean, and I think a combinations of macros and a gateway mock.rs to automatically download and compress fgw responses would be cleaner (would also be nice as I have seen other areas of the codebase rely either on json or manually typing large structs fields). Imo this works until a better system is in place.